### PR TITLE
[FEATURE] Helper functions for eth wallets

### PR DESCRIPTION
**Pull request by moldis**
The branch contains helping functions to generate a wallet based on ETH wallet. It will be compiled to gomobile later on.

### CODE REVIEW

File: core/zcncrypto/ed255190chain.go [hunk 1]
```diff
@@ -57,6 +57,11 @@ func (ed *ED255190chainScheme) GenerateKeys() (*Wallet, error) {
 	return w, nil
 }
 
+//GenerateKeysWithEth - not implemented
+func (ed *ED255190chainScheme) GenerateKeysWithEth(mnemonic, password string) (*Wallet, error) {
+	return nil, nil
```
Comment (Dmdv) [inline core/zcncrypto/ed255190chain.go:62]: I would return an error so not to confuse the client caller

File: core/zcncrypto/bls0chain.go [hunk 2]
```diff
@@ -31,8 +31,26 @@ func NewBLS0ChainScheme() *BLS0ChainScheme {
 	return &BLS0ChainScheme{}
 }
 
+func (b0 *BLS0ChainScheme) GenerateKeysWithEth(mnemonic, password string) (*Wallet, error) {
```
Comment (Dmdv) [inline core/zcncrypto/bls0chain.go:34]: Missing test for this new function

File: zcncore/ethwallet.go [hunk 3]
```diff
@@ -0,0 +1,301 @@
+package zcncore
```
Comment (Dmdv) [inline zcncore/ethwallet.go:1]: go test coverage 0.3% - could you pls add more tests?

**Review by Dmdv — CHANGES_REQUESTED** [review]

File: zcncore/ethwallet.go (hunk 3 above)
Comment (moldis) [reply zcncore/ethwallet.go:1]: my latest commit is 67%

**Review by Sriep — APPROVED** [review]

**Pull request merged**